src/Symfony/Component/Validator/Constraints/Finite.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ *
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Guillaume Aveline <guillaume@codr.fr>
+ */
+#[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
+class Finite extends Constraint
+{
+    public const NOT_FINITE_ERROR = '5f809eb0-78b9-492d-ad37-5a5188390415';
+
+    protected const ERROR_NAMES = [
+        self::NOT_FINITE_ERROR => 'NOT_FINITE_ERROR',
+    ];
+
+    public $message = 'This value should be finite.';
```

Comment:
Just asking if we should make this error message a bit more precise.

`This value should be finite.` -> `This value should be a finite number.`

Maybe we should **not** do this, because the error messages of equivalent constraints are also like this:

`This value should be positive.` (instead of `This value should be a positive number.`)